front/src/applications/operationalStudies/components/BreadCrumbs.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+type Props = {
+  projectName?: string;
+  studyName?: string;
+  scenarioName?: string;
+};
+
+export default function BreadCrumbs({ projectName, studyName, scenarioName }: Props) {
+  const { t } = useTranslation('operationalStudies/project');
+  return (
+    <div className="navbar-breadcrumbs">
+      {!projectName && !studyName && !scenarioName ? (
```

Comment:
you can probably simplifies like this:

```
if (projectName) {
 // logic using only studyName and scenarioName
  return ()
  }

return ()
```

Reply:
Yes ! already proposed by @clarani but not so easy, @Math-R is on the breach ;)